docs/source/other_models/pregnancy/index.rst

```diff
@@ -118,6 +118,9 @@ We will model pregnancy as a characteristic of women of reproductive age in our
   * - p
     - Pregnant
     - 
+  * - md
+    - Maternal disorders 
+    - Simulated state that lasts for single timestep in order to accrue maternal disorders YLDs - not representative of biological state. Can be with (infected/md_i) or without (susceptible/md_s) maternal disorders while within this state according to the maternal disorders cause model document.
```

Comment:
I think infected and susceptible will lead to confusion here. What do you think about md_w and md_wo for with and without maternal disorders?

Reply:
Good thinking, I've made the update.